src/main/java/frc/trigon/robot/commands/CommandConstants.java

```diff
@@ -95,6 +97,13 @@ public class CommandConstants {
             DEFAULT_AMP_ALIGNER_COMMAND = AmpAlignerCommands.getSetTargetStateCommand(AmpAlignerConstants.AmpAlignerState.CLOSE),
             SHOOT_AT_SPEAKER_COMMAND = ShootingCommands.getShootAtShootingTargetCommand(false),
             DELIVERY_COMMAND = ShootingCommands.getShootAtShootingTargetCommand(true),
+            WHEEL_RADIUS_CHARACTERIZATION_COMMAND = new WheelRadiusCharacterizationCommand(
+                    SwerveConstants.MODULE_LOCATIONS,
+                    () -> Arrays.stream(RobotContainer.SWERVE.getDriveWheelPositions()).mapToDouble(Rotation2d::getRadians).toArray(),
```

Comment:
And make the lib function accept an array of rotation2d
Going with just radians is also okay, but that's technically more correct